phpcs.xml

```diff
@@ -86,7 +86,8 @@
 
 	<!-- Use the PHPCompatibility Ruleset -->
 	<config name="testVersion" value="5.6-99.0"/>
-	<rule ref="PHPCompatibilityWP">
-		<include-pattern>*\.php$</include-pattern>
-	</rule>
-</ruleset>
+	<rule ref="PHPCompatibilityWP" />
```

Comment:
Just wondering why the `<include-pattern>` is being removed.

I did see that the extensions are limited to `php`, but considering WPThemeReview also contains some non-php checks, removing that and leaving this would make more sense.